crawler.py

```diff
@@ -87,15 +89,26 @@ def test_crash(driver):
 var crash = badptr.contents;""")
 
 
-def get_domain_list(n_sites):
+def get_domain_list(n_sites, exclude_option):
     """Get the top n sites from the tranco list"""
-    tr = Tranco(cache=False)
-
-    if not n_sites:
-        n_sites = DEFAULT_NUM_SITES
```

Comment:
Looks like we're still missing the `DEFAULT_NUM_SITES` fallback. I think this happened because we incorrectly removed an older version of the fallback in https://github.com/EFForg/badger-sett/pull/51/commits/628c7ab39fc8b735da4f2d6c78c1d963f5e9a543#diff-ee054b4b797f6c2a5743d2654764a8edL96-L97. If `n_sites` is falsy, we should initialize it to 2000 (before the merge) / `DEFAULT_NUM_SITES` (after the merge).

Reply:
Ah thank you! Added this.